hasher-matcher-actioner/hmalib/lambdas/hashing.py

```diff
@@ -48,15 +53,23 @@ def get_sqs_client() -> SQSClient:
 DYNAMODB_TABLE = os.environ["DYNAMODB_TABLE"]
 IMAGE_PREFIX = os.environ["IMAGE_PREFIX"]
 BANKS_TABLE = os.environ["BANKS_TABLE"]
+HMA_CONFIG_TABLE = os.environ["HMA_CONFIG_TABLE"]
 
 
-# If you want to support additional content or signal types, they can be added
-# here.
-hasher = UnifiedHasher(
-    supported_content_types=[PhotoContent, VideoContent],
-    supported_signal_types=[PdqSignal, VideoMD5Signal],
-    output_queue_url=OUTPUT_QUEUE_URL,
-)
+@functools.lru_cache(maxsize=None)
+def _get_signal_type_mapping() -> HMASignalTypeMapping:
+    """
+    Cache-get signalTypeMapping. Call only if HMAConfig has been initialized.
+    """
+    return get_pytx_functionality_mapping().signal_and_content
+
+
+@functools.lru_cache(maxsize=None)
```

Comment:
Is this the default?

Reply:
No, maxsize defaults to 128. Using the sentinel value None disables the lru counting so saves a tiny bit of resources.